packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -4,15 +4,11 @@
 
 ```ts
 
+import type { EventEmitter } from 'events_pkg';
 import { FieldSchemaUnsafe as FieldSchemaUnsafe_2 } from './typesUnsafe.js';
-import { FluidObject } from '@fluidframework/core-interfaces';
-import { IChannel } from '@fluidframework/datastore-definitions';
-import type { IErrorBase } from '@fluidframework/core-interfaces';
-import { IEvent } from '@fluidframework/core-interfaces';
-import { IEventProvider } from '@fluidframework/core-interfaces';
-import { IFluidHandle } from '@fluidframework/core-interfaces';
-import { IFluidLoadable } from '@fluidframework/core-interfaces';
-import { ISharedObjectKind } from '@fluidframework/shared-object-base';
+import { FluidObject as FluidObject_2 } from '@fluidframework/core-interfaces';
+import { IFluidHandle as IFluidHandle_2 } from '@fluidframework/core-interfaces';
+import { IFluidLoadable as IFluidLoadable_2 } from '@fluidframework/core-interfaces';
```

Comment:
This is strange/broken. API extractor bug? Rollup issue?

Reply:
See https://github.com/microsoft/rushstack/issues/4654

Reply:
This does seem problematic (wonder how it would affect generated doc pages).  Any workaround?  If we enforce consistently importing from /internal everywhere in our repo for example?